.openpublishing.redirection.json

```diff
@@ -1683,6 +1683,14 @@
             "redirect_url": "/dotnet/standard/native-interop/qualify-net-types-for-interoperation",
             "redirect_document_id": true
         },
+        {
+            "source_path": "docs/framework/migration-guide/retargeting/index.md",
+            "redirect_url": "/dotnet/framework/migration-guide/application-compatibility"
+        },
+        {
+            "source_path": "docs/framework/migration-guide/runtime/index.md",
+            "redirect_url": "/dotnet/framework/migration-guide/application-compatibility"
+        },
         {
             "source_path": "docs/framework/migration-guide/application-compatibility-in-the-net-framework-4-5-1.md",
             "redirect_url": "/dotnet/framework/migration-guide/application-compatibility"
```

Comment:
The redirections are ordered alphabetically, so `docs/framework/migration-guide/retargeting` and `docs/framework/migration-guide/runtime` should come after `docs/framework/migration-guide/release-keys-and-os-versions.md`

Reply:
There's a [sorting script in the VS repo](https://github.com/MicrosoftDocs/visualstudio-docs/blob/master/sort-redir.py). We could set up an Azure Pipeline to run the script every time the file is changed, or on a scheduled basis.

Reply:
I really like this idea. What do you think @mairaw @Thraka ?

Reply:
I filed an issue #15577

Reply:
I like that idea too! But adding them in the right order is what helps to prevent the merge conflicts. Otherwise, if two PRs are targeting the same file and both are just appending at the end and the script fixes that, you'll still have to resolve the conflict no? We could definitely test that.

Reply:
I *think* it would work. It does require testing and confirmation.